src/App.tsx

```diff
@@ -16,7 +16,31 @@ import httpClient from "./httpClient";
 import { LanguageType, PageStateEnum } from "./types";
 import { initializeData } from "./utils/stateUpdateUtils";
 import { setLanguageType } from "./utils/translate/translateService";
-import { ANALYZE_URL } from "./constants";
+import { ANALYZE_URL, CANADA_URL } from "./constants";
+
+function CanadianTableau() {
+  return (
+    <TableauEmbed
+      url={CANADA_URL}
+      options={{
+        width: "80vw",
+        height: "4100px"
+      }}
+    />
+  )
+}
+
+function AnalyzeTableau() {
+  return (
+    <TableauEmbed
+      url={ANALYZE_URL}
+      options={{
+        width: "80vw",
+        height: "3200px"
```

Comment:
have these sizes been tested on mobile?

Reply:
They look okay ish, but I think the precise heights/widths for mobile rendering are also defined in tableau so I will sync with jordan on that and push changes accordingly